src/org/zaproxy/zap/extension/pscanrules/CookieUtils.java

```diff
@@ -163,4 +167,19 @@ private static void validateParameterNotNull(Object parameter, String name) {
             throw new IllegalArgumentException("The parameter " + name + " must not be null.");
         }
     }
+    
+    public static Set<String> getCookieIgnoreList(Model model) {
```

Comment:
Maybe this should use a Configuration instead of the Model? Less things that need to be mocked in the tests.